feature-staking-impl/src/main/res/layout/view_alert.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
```

Comment:
Minor stuff - This wont be applied to merge tag, so it is better to directly indicate it using `tools:` namespace

Reply:
Done